src/back/deps/s2j.Dockerfile

```
@@ -0,0 +1,10 @@
+FROM python:3.10
+
+RUN mkdir ./swagger2jsonschema
+COPY doc/api/api.yaml ./swagger2jsonschema
+
+WORKDIR /swagger2jsonschema
```

Проверь пожалуйста будет ли так работать:
```suggestion
WORKDIR /swagger2jsonschema
COPY doc/api/api.yaml .
```

поидее WORKDIR должна будет сама создать папку

---

Да, работает, поправил, вливаю.